app/js/components/manageApps/ManageApps.jsx

```diff
@@ -415,6 +417,30 @@ export default class ManageApps extends React.Component {
     };
   }
 
+  checkForUpdates() {
+    const installedAddons = this.state.appList;
+    const updatesAvailable = [];
+    axios.get(`https://addons.openmrs.org/api/v1//addon`).then((response) => {
+      this.state.appList.forEach((addon) => {
+        response.data.forEach((result) => {
+          if (addon.appDetails.name == result.name) {
+            result.latestVersion > addon.appDetails.version ?
+              updatesAvailable.push(addon.appDetails.name)
+              :
+              null
+            return;
+          }
+        });
+      });
+
+      this.setState({ updatesAvailable });
```

Comment:
Which value are you setting updatesAvailable to?

Reply:
This is ES6 shorthand for this.setState({ updatesAvailable: updatesAvailable});

Reply:
👍